exonum_client/protobuf_loader.py

```diff
@@ -100,10 +103,12 @@ def __init__(self, client: Optional[ProtobufProviderInterface] = None):
         # Check that the client is the same as expected:
         if ProtobufLoader._entity is not None:
             if client is not None and client != ProtobufLoader._entity.client:
+                logging.critical("Attempt to create ProtobufLoader entity with a different client.")
```

Comment:
Here you can provide info about used and provided clients, including the host/ports and the object address as well (to distinguish two different client objects created for the same host).